src/System.Globalization.Calendars/tests/CalendarTestWithConfigSwitch/xunit.console.netcore.runtimeconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "runtimeOptions": {
+    "configProperties": {
+      "Switch.System.Globalization.EnforceJapaneseEraYearRanges": true
```

Comment:
are you sure the tests with this kind of the config switches work fine with UWP? I guess so, but I wanted to confirm this is working fine.

Reply:
I don't think this test even gets compiled in sicne it isn't included in the System.Globalization.Calendars.Tests.csproj file.  That must have come in a different commit than the one I cherry picked.  Since I don't think this test is actually being used, it's probably simpler to leave it out of this PR.  Will update momentarily.